### ISSUE TITLE
Updates for geoip databases?

### ISSUE THREAD

**Issue by gravitystorm** [openstreetmap/chef#333]
I was looking through the code for deploying the maxminddb instances used by the rails port, when I came across this line in the geoipupdate section:

https://github.com/openstreetmap/chef/blob/5feb351746f786067f394ae5cc2e9ba58aa397ce/cookbooks/geoipupdate/recipes/default.rb#L40

As I read it, this means that a rails server will get the required databases, but never update them. Is my understanding correct?

If so, would it be possible to refresh the databases every now and then?

**Comment by tomhughes** [comment]
Well that's not actually the current code because the path has been changed since then.

But that was only ever intended to populate the initial database - the `geoipupdate` packages contains a cron script that updates it once a week after that.

At least it did when I wrote that, back when we were using the Ubuntu package. Since then we switched to the maxmind PPA version which apparently doesn't include it, except that because it was marked as a config file in the Ubuntu package it survives over an upgrade to the maxmind version!

So newly installed machines had no cron jobs but older machines which had been updated did...

I have now added a bunch of stuff to clean things up, and a new timer unit to do weekly updates.
